pkg/influx/query/query.go

```diff
@@ -0,0 +1,50 @@
+package query
+
+import (
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/valyala/fasthttp"
+)
+
+func GetQueryHandler(ctx *fasthttp.RequestCtx, myid uint64) {
+
+	q := string(ctx.QueryArgs().Peek("q"))
+	if strings.ToLower(q) == "show databases" {
+		databaseResponse := `{
```

Comment:
make this as a const at the top of file, so that we dont have to recreate this memory

Reply:
name it `INFLUX_SHOW_DB_RESP`